src/Eloquent/Model.php

```diff
@@ -2,570 +2,19 @@
 
 namespace Jenssegers\Mongodb\Eloquent;
 
-use function array_key_exists;
-use DateTimeInterface;
-use function explode;
-use Illuminate\Contracts\Queue\QueueableCollection;
-use Illuminate\Contracts\Queue\QueueableEntity;
-use Illuminate\Contracts\Support\Arrayable;
 use Illuminate\Database\Eloquent\Model as BaseModel;
-use Illuminate\Database\Eloquent\Relations\Relation;
-use Illuminate\Support\Arr;
-use Illuminate\Support\Facades\Date;
-use Illuminate\Support\Str;
-use function in_array;
-use Jenssegers\Mongodb\Query\Builder as QueryBuilder;
-use MongoDB\BSON\Binary;
-use MongoDB\BSON\ObjectID;
-use MongoDB\BSON\UTCDateTime;
-use function uniqid;
 
-abstract class Model extends BaseModel
-{
-    use HybridRelations, EmbedsRelations;
+$connection = config('database.default');
```

Comment:
Why don't use eloquent model for MySQL for example directly, this package is created for work with mongodb, not different drivers

Reply:
this way the package acts like a plugin to add the ability for the developer to use MongoDB.
think of it as a laravel with MongoDB compatibility.